ksql-engine/src/test/java/io/confluent/ksql/QueryTranslationTest.java

```diff
@@ -104,23 +105,50 @@ public void shouldBuildAndExecuteQueries() {
   }
 
   @Parameterized.Parameters(name = "{0}")
-  public static Collection<Object[]> data() throws IOException {
-    final String topologyVersion = System.getProperty(TOPOLOGY_VERSION_PROP, CURRENT_TOPOLOGY_VERSION);
-    final String topologyDirectory = TOPOLOGY_CHECKS_DIR + "/" + topologyVersion;
-    final Map<String, TopologyAndConfigs> expectedTopologies = loadExpectedTopologies(topologyDirectory);
+  public static Collection<Object[]> data() {
+    final String[] topologyVersions =
+        System.getProperty(TOPOLOGY_VERSIONS_PROP, CURRENT_TOPOLOGY_VERSIONS)
+            .split(TOPOLOGY_VERSIONS_DELIMITER);
+    final List<TopologiesAndVersion> expectedTopologies = loadTopologiesAndVersions(topologyVersions);
```

Comment:
nit: i'd find it a bit more readable if we called this from buildVersionedTestCases. Then the code here could just be:

```
return buildTestCases()
    .flatMap(q -> buildVersionedTestCases(q, topologyVersions))
    .map(...)
    .collect(...)
```

Reply:
If we moved the call to `loadTopologiesAndVersions()` into `buildVersionedTestCases()`, then it'd be called once for each test case, right? That seems undesirable. Compromised by moving the lines above into a separate function, so it's hopefully more readable.